lua/ulx/modules/sh/timed_gag.lua

```diff
@@ -0,0 +1,98 @@
+CATEGORY_NAME = "CFC"
+SQL_TABLE = "cfc_timed_gags"a
+
+local GaggedPlayers = {}
+
+local function createTable()
+    if sql.TableExists( SQL_TABLE ) then return end
+
+    local createTableQuery = string.format("CREATE TABLE %s(steam_id TEXT, expiration BIGINT)", SQL_TABLE)
```

Comment:
Ugh I'm like 80% sure that someone made an sql abstraction. Might've been the maker of ULX? Just don't remember